tests/TopN.CoreFX.issues.json

```diff
@@ -0,0 +1,5 @@
+[
+    "System.Collections.Tests",
+    "System.Collections.Immutable.Tests",
+    "System.Net.Security.Tests"
```

Comment:
Why `System.Net.Security.Tests` ?

Reply:
All three are placeholders. System.Net.Security.Tests has an explicit dependenency on CoreFX.Private.TestUitilities, so that's why I included it. 
The actual tests need to be decided.